tests/test_urfkill.py

```diff
@@ -73,11 +73,19 @@ def test_block(self):
         property_object = bus.get_object('org.freedesktop.URfkill', '/org/freedesktop/URfkill/WLAN')
         propiface = dbus.Interface(property_object, 'org.freedesktop.DBus.Properties')
         self.assertEqual(propiface.Get('org.freedesktop.URfkill.Killswitch', 'state'), 0)
-        iface.Block(1, True)
+
+        ret_block = iface.Block(1, True)
+        self.assertTrue(ret_block)
```

Comment:
the variable is redundant, please just do

     self.assertTrue(iface.Block(...))